modules/labs-react/toast/README.md

```diff
@@ -0,0 +1,178 @@
+# Canvas Kit Toast
+
+<a href="https://github.com/Workday/canvas-kit/tree/master/modules/labs-react/README.md">
+  <img src="https://img.shields.io/badge/LABS-alpha-orange" alt="LABS: Alpha" />
+</a>  This component is work in progress and currently in pre-release.
+
+Compound component implementation of the existing Toast component. Canvas-styled React toast component. Allows for a brief message to be be shown about a process or
+action occurring in the app.
```

Comment:
```suggestion
Toasts contain low-emphasis updates or messages about the status of an application’s process.
```
Nit: I'm going to [borrow language](https://canvas.workdaydesign.com/components/popups/toast/) from our design site instead.

Reply:
I ended up wiping out most of the README.md as is convention after migrating a component's docs to MDX. I did include this language in the MDX, however.